### ISSUE TITLE
Ability to copy entry when method is "Only save entries to the site they were created in"

### ISSUE THREAD

**Issue by howells** [Goldinteractive/craft3-sitecopy#30]
I see that SiteCopy is disabled when a section's propagation method is "Only save entries to the site they were created in", but wondered why this was and whether this option might be added? I have a use case where I need to basically duplicate the entire content of a template site, but then keep the content of the sections quite separate.

**Comment by christianruhstaller** [comment]
Yes that is the expected behavior as the plugin just can copy to entries which already exists. If you have enabled the setting "Only save entries to the site they were created in" Craft does not create an entry for it and it actually does not exist.
If we would override this behavior we would need to create our own logic for propagating which we would like to prevent.

As I understand your use case should it not be enough to just enable the field translations? Can you give use a more detailed explanation of the wanted workflow and use case? Maybe there exists other solutions or feature updates which are more streamlined with the Craft workflow.

**Comment by howells** [comment]
@christianruhstaller thanks, yeah that makes sense. In this case there is a page that's used as an example template, which may or may not be on the target site, and might just be deleted entirely from that site. I've found that the [Site Duplicate](https://plugins.craftcms.com/siteduplicate) plugin seems to do what I need, but isn't in active development as far as I can tell.

**Comment by christianruhstaller** [comment]
Ok, well it sounds as a good addition to the current feature. We will have a look at it but we can't say when it will be going to be implemented as our resources are quite limited currently.

**Comment by howells** [comment]
@christianruhstaller apologies, this plugin _doesn't_ actually copy the entry if it doesn't exist, I just checked with a clean new site. But that would be a great feature!

**Comment by howells** [comment]
@christianruhstaller hey! Just wondering if you reckon this feature might be added any time soon? It's somewhat critical for a site I'm working on right now. No worries if not, I'll have to work something else out.
